hadoop-ozone/dev-support/checks/acceptance.sh

```diff
@@ -29,6 +29,7 @@ fi
 
 cd "$DIST_DIR/compose" || exit 1
 ./test-all.sh
+RES=$?
 cp results/* "$REPORT_DIR/"
 cp "$REPORT_DIR/log.html" "$REPORT_DIR/summary.html"
-exit $?
+return $RES
```

Comment:
bash says: `can only 'return' from a function or sourced script` (exit code of `acceptance.sh` would be 1 if executed directly instead of sourcing it)